.gitmodules

```diff
@@ -338,6 +338,9 @@
 [submodule "contrib/liburing"]
 	path = contrib/liburing
 	url = https://github.com/axboe/liburing
+[submodule "contrib/libfiu"]
+	path = contrib/libfiu
+	url = https://github.com/hanfei1991/libfiu.git
```

Comment:
Can we just reference upstream? (if patches are necessary, the convention is to make a fork in the ClickHouse umbrella organization on GitHub)

Reply:
yep, a fork in ch should be approved by Alexey. Let's do it when the pr is approved